pkg/config/config.go

```diff
@@ -0,0 +1,75 @@
+package config
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+)
+
+// File reflects the config file.
+// Specification: https://github.com/notaryproject/notation/pull/76
+type File struct {
+	VerificationCertificates VerificationCertificates `json:"verificationCerts"`
+	SigningKeys              SigningKeys              `json:"signingKeys,omitempty"`
+	InsecureRegistries       []string                 `json:"insecureRegistries"`
```

Comment:
instead of exporting types can we export routines so that underlying implementation can be changed without affecting callers.